start-local.sh

```diff
@@ -474,12 +501,21 @@ ES_LOCAL_CONTAINER_NAME=$elasticsearch_container_name
 ES_LOCAL_PASSWORD=$es_password
 ES_LOCAL_PORT=9200
 ES_LOCAL_URL=http://localhost:\${ES_LOCAL_PORT}
-ES_LOCAL_HEAP_INIT=128m
-ES_LOCAL_HEAP_MAX=2g
 ES_LOCAL_DISK_SPACE_REQUIRED=1gb
 EOM
 
-  if  [ -z "${esonly:-}" ]; then
+  if [ "$otel" = "true" ]; then
+    cat >> .env <<- EOM
+ES_LOCAL_JAVA_OPTS="-Xms2g -Xmx2g"
```

Comment:
Do we set that Xms to 2GB intentionally? I don't think we'll compare well to Grafana's stack and others like that ("big", "bloated",...). We should IMO still initialize as small as possible with the necessary room to grow if needed. Even if we take a small performance hit when needing to increase the heap size.

Reply:
Yeah, IMO this can (should) use the same settings and general approach used for the other options. We still want this to start really lightweight.